frontend_lib/src/component/Timeline/Timeline.jsx

```diff
@@ -138,25 +138,21 @@ class Timeline extends React.Component {
 
             <div className={classnames(`${props.customClass}__texteditor__wrapper`, 'timeline__texteditor__wrapper')}>
               <div className={classnames(`${props.customClass}__texteditor__advancedtext`, 'timeline__texteditor__advancedtext')}>
-                <button
-                  type='button'
+                <div
```

Comment:
A button tag most important thing is that it is something clickable.
It's design is secondary.
Semantically, it's better to keep a button tag and adapt it's design if required.
I think it would be best to keep the button tag and remove the .btn and .outlineTextBtn class names